src/actions/DispensaryActions.js

```diff
@@ -20,15 +20,15 @@ const switchDataSet = () => ({ type: DISPENSARY_ACTIONS.SWITCH });
 
 const refresh = () => ({ type: DISPENSARY_ACTIONS.REFRESH });
 
-const lookupRecord = () => ({ type: DISPENSARY_ACTIONS.LOOKUP_RECORD });
+const openLookupModal = () => ({ type: DISPENSARY_ACTIONS.OPEN_LOOKUP_MODAL });
 
-const cancelLookupRecord = () => ({ type: DISPENSARY_ACTIONS.CANCEL_LOOKUP_RECORD });
+const closeLookupModal = () => ({ type: DISPENSARY_ACTIONS.CLOSE_LOOKUP_MODAL });
```

Comment:
Haven't tested but possibly need other cleanup here such as resetting the form state?

Reply:
Yeah, currently it's always being called as:

```
batch(() => {
  dispatch(FormActions.resetForm());
  dispatch(DispensaryActions.closeLookupModal());
});
```

Good point, should just be included in the action.

Reply:
197b1cd1.